### ISSUE TITLE
using fully qualified class names in inheritance is wrongly reported as inheritance across namespaces

### ISSUE THREAD

**Issue by duritong** [rodjek/puppet-lint#148]
The following should give 0 warnings:

```
$ cat test.pp 
# I'm a super class
class foo::super { }
# I'm a subclass
class foo::sub inherits foo::super {}
$ puppet-lint --no-autoloader_layout-check test.pp
WARNING: class inherits across namespaces on line 4
```
